lib/travis/build/script/r.rb

```diff
@@ -84,7 +84,9 @@ def setup
             sh.cmd 'sudo chmod 2777 /usr/local/lib/R /usr/local/lib/R/site-library'
 
           when 'osx'
-            sh.cmd 'brew update', retry: true
+            # We want to update, but we don't need the 800+ lines of
+            # output.
+            sh.cmd 'brew update >/dev/null', retry: true
```

Comment:
oh -- in the generated bash script, it looked like `echo` only controlled whether or not we printed the command before executing it. does it also prevent output from landing in the logs?